evaluation.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+from hourglass import HourglassNet
+from data_generator import DataGenerator
+from constants import *
+from util import *
+
+import time
+from datetime import timedelta
+import matplotlib.pyplot as plt
+import numpy as np
+from PIL import Image
+import glob
+import cv2
+
+def predict(hourglass, model_json, model_weights, subset):
+    hgnet = HourglassNet(num_classes=NUM_COCO_KEYPOINTS, num_stacks=hourglass, num_channels=NUM_CHANNELS, inres=INPUT_DIM,
+                            outres=OUTPUT_DIM)
+    hgnet._load_model(model_json, model_weights)
+
+    val_df = hgnet.load_and_filter_annotations(DEFAULT_TRAIN_ANNOT_PATH, DEFAULT_VAL_ANNOT_PATH, subset)
```

Comment:
`load_and_filter_annotations` returns `train_df, val_df` so for now we would want to take the second df. Also, we will probably wanna modify `load_and_filter_annotations` to return 3 dfs, `train_df, val_df, test_df` and we would use `test_df` here. I don't think subset argument is necessary here unless we wanna to evaluations on a subset of the training set.

Reply:
We may just want to `load_and_filter_annotations` to optionally return test, since we don't want to download test2017.zip on the colab VM